config/config.go

```diff
@@ -0,0 +1,38 @@
+package config
+
+import (
+	"os"
+	"strconv"
+)
+
+var (
+	SERVER_PORT    string
+	SERVER_FILEDIR string
+	DB_FILE        string
+	PASSWORD       string
+	SECRETKEY      string = "top-secret"

```

Comment:
По логике названий переменных выше должно быть
```suggestion
	SECRET_KEY      string = "top-secret"

```